### DISCUSSION TITLE
Python view on lxplus supporting zfit

### DISCUSSION THREAD

**Discussion by johnmcgowan1** [zfit/zfit#327] [Q&A]
Hello,

I am working on a package that I would like others to use easily on lxplus, specifically for ATLAS. Is there an existing view that has all the packages with compatible versions to run zfit? Or a simple description of setup steps to take on lxplus? I have not found a view in which I can use uproot and zfit successfully.

Thank you,
John

**Comment by jonas-eschle** [comment]
Hi John,

That sounds quite good, which package are you working on? I am not quite familiar with the "view" that you mentioned, but zfit and uproot should not be a problem to have installed on the same machine as zfit actually relies on uproot (version 4). I would recommend to [install zfit with pip](https://github.com/zfit/zfit#installing).

What exactly do you mean by the view? Or did that maybe already resolve the issue?

Best,
Jonas

**Reply by jonas-eschle** [reply]
Hi @johnmcgowan1, I haven't heard back, did you manage to run it?

**Comment by jonas-eschle** [comment]
Hi @johnmcgowan1 , it should be installable with pip (if there are any issues, let me know!) but there is also an [LCG](https://lcginfo.cern.ch/) with a quite recent zfit version
